public/js/Histogram.js

```diff
@@ -0,0 +1,95 @@
+var d3 = require('d3');
+var _ = require('lodash');
+_.mixin(require('lodash-deep'));
```

Comment:
The creators of underscore also should be shot. The point is that an underscore is in no way a good package name. Might as well release "space", where it is imported as some unicode whitespace character that isn't ignored by parsers. Then it's all ` .function()` everywhere. FFS!

I get what they were going for, they were saying, "our library is going to be so fucking important that it won't need context, readable namespacing, or any of that stuff." Well, that's just arrogant and massively hurts readability.

What's `lodash-deep`? Does it somehow pollute the namespace even more? =P